bin/create-hierarchy

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env python3
```

Comment:
Is this script replaced by `check-locations`? I don't see this being used anywhere.

Reply:
Oops, you're right. it's no longer being used anywhere.